README.md

```diff
@@ -123,69 +49,288 @@ GRANT ALL PRIVILEGES ON DATABASE sample TO manager;
 | SENDGRID_FROM_EMAIL               | <sendgridfromemail>  | The e-mail address used for notifications   |
 | LOCAL                             | False                | Tells the site to use external API or mocks |
 
+# Getting Started
+
+## First time here?
+
+The docker image for this branch is created differently from the other branches (`python manage.py runserver 0.0.0.0` is now run inside the image, not as a command on the image in the `docker-compose.yml`). This means that the old containers and images for other branches need to be deleted so the new one can be built.
+
+```
+$ docker ps --all
+CONTAINER ID     IMAGE               COMMAND                  CREATED        STATUS                  PORTS       NAMES
+<container_id>   wagtail-cms_web     "python manage.py ru…"   3 days ago     Exited (0) 3 days ago               wagtail-cms_web_1
+............     .....               "...................."   ..........     .............           27017/tcp   .................
+$ docker rmi -f <container_id>
+$ docker images
+REPOSITORY                            TAG          IMAGE ID       CREATED         SIZE
+wagtail-cms_web                       latest       <image_id>     11 days ago     1.07GB
+...............                       latest       ............   3 weeks ago     ......
+$ docker rm <image_id>
+...
+```
+
+This now means when you run `docker-compose up` docker will rebuild the image using the new commands in the `Dockerfile`, and then build a new container from that
+
+> *REMEMBER:* To run the commands above when switching back to the other branches on the project
+
+## Tests
+
+```
+$ docker container exec -it wagtail-cms_web_1 python manage.py test
+```
+
+> *IMPORTANT:* When fixing an issue write a test first to prove it's broken. Then, and only then, fix the problem
+
+## Configure
 
-### Running the site
+### Secret Key
 
-#### Running without Docker
+```
+$ python3
+>>> import secrets
+>>> chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
+>>> ''.join(secrets.choice(chars) for i in range(50))
+<secret_key>
+>>> exit()
+$ vim docker-compose.yml
+...
+SECRET_KEY: "<secret_key>"
+...
+```
 
-Create a Python 3.6.8 virtual environment and run the following commands from the root directory of the project:
+### Defaults
 
 ```
-pip install -r requirements.txt
-python manage.py migrate
-python manage.py createsuperuser
-python manage.py runserver
+$ cp docker-compose.yml.example docker-compose.yml
+$ vim docker-compose.yml
+...
+DBHOST:               "db"
+DBNAME:               "django"
+DBUSER:               "docker"
+DBPASSWORD:           "docker"
+...
+
+...
+MONGODB_HOST:         "mongo"
+MONGODB_USERNAME:     "mongodb"
+MONGODB_PASSWORD:     "mongodb"
+...
+SEARCHAPIHOST:        "..."
+...
+# docker-compose up
```

Comment:
Is this supposed to be a title? If so, it should be pushed down 1 line as it's currently within a code block